build

```diff
@@ -94,6 +94,10 @@ MCMD=make
 elif [[ $platform == 'macosx' ]]; then
   QCMD=qmake
   QSPEC=macx-clang
+  arch=`arch`
```

Comment:
How does this work, if you define arch as `arch` and then check if it's `arm64`? Why isn't the architecture set automatically?

Reply:
There's an `arch` command (`/usr/bin/arch`) which returns architecture, similar to how we get [platform name](https://github.com/jacktrip/jacktrip/blob/main/build#L71).

> Why isn't the architecture set automatically?

I was building off of how the script was already set up with explicitly set architecture. It seems we can omit it, at least on macos. I'll update the PR.

Reply:
What the… Why this syntax?